### ISSUE TITLE
user listing pagination error

### ISSUE THREAD

**Issue by mikehelmick** [google/exposure-notifications-verification-server#805]
### TL;DR

Noticed - if there are 3 pages of users, the buttons show `previous, 1, 2 , next`

![image](https://user-images.githubusercontent.com/92319/95801078-115bbe00-0cae-11eb-8792-8914c7539081.png)

The 3rd (last) page can only be reached by clicking `2` and then clicking `next`.

Page 1 (0-25 of 56)

![image](https://user-images.githubusercontent.com/92319/95801115-305a5000-0cae-11eb-964d-d2516ea25ef5.png)

Page 2 (25-50 of 56)

![image](https://user-images.githubusercontent.com/92319/95801129-3d773f00-0cae-11eb-8ecf-a77d7f76d0d2.png)

And then 'next' is doesn't handle the offset correctly

31-56 of 56 - should be 51-56 of 56

![image](https://user-images.githubusercontent.com/92319/95801158-5122a580-0cae-11eb-85f1-57938fef7d0a.png)

/kind bug
